### ISSUE TITLE
`Uncrossed()` is improperly handled

### ISSUE THREAD

**Issue by ike709** [wixoaGit/OpenDream#821]
See this TG code:
```
/atom/movable/Uncrossed(atom/movable/uncrossed_atom)
    SHOULD_NOT_OVERRIDE(TRUE)
    CRASH("/atom/movable/Uncrossed() was called")
```

We call `Uncrossed()` in `Exited()` which triggers this crash

**Comment by ike709** [comment]
TG has `/atom/Exited(atom/movable/gone, direction)` overridden to not call parent, I wonder if we just need to move this code to a higher level type and have it check if `src` is a turf

**Comment by wixoaGit** [comment]
> TG has `/atom/Exited(atom/movable/gone, direction)` overridden to not call parent, I wonder if we just need to move our Exited() code to a higher level type and have it check if `src` is a turf

This doesn't seem to be the case. Overriding `/atom/Exited()` with an empty proc in BYOND leads to `/atom/movable/Uncrossed()` still being called.

**Comment by wixoaGit** [comment]
Fixed by 813fca3a676dfa276bb4ccc44209d110382e1b43
